src/docs/spec.md

```diff
@@ -370,11 +372,13 @@ We recommend to use the following *filename conventions* for SSSOM metadatafiles
 Example ([download](https://raw.githubusercontent.com/mapping-commons/sssom/master/examples/external/mp-hp-exact-0.0.1.sssom.yml)):
 
 ```
-creator_id: "https://orcid.org/0000-0002-7356-1779"
 curie_map: 
   HP: "http://purl.obolibrary.org/obo/HP_"
   MP: "http://purl.obolibrary.org/obo/MP_"
-  skos: "http://www.w3.org/2004/02/skos/core"
+  skos: "http://www.w3.org/2004/02/skos/core#"
+  orcid: "https://orcid.org/"
```

Comment:
Since the curie map should be in alphabetical order, `orcid` should be moved after `MP`.